src/sinks/prometheus/exporter.rs

```diff
@@ -47,6 +48,7 @@ use crate::{
 };
 
 const MIN_FLUSH_PERIOD_SECS: u64 = 1;
+const SUPPORTED_AUTHENTICATION_SCHEMAS: HeaderValue = HeaderValue::from_static("Basic, Bearer");
```

Comment:
Ah, I see, I guess this can't actually be a constant like I thought it could. You can just use `from_static` in-line below.

Reply:
Fixed.